### ISSUE TITLE
[Bug Report][3.7.6] v-main wrong padding when using navigation drawer location top

### ISSUE THREAD

**Issue by apodacaduron** [vuetifyjs/vuetify#20811]
### Environment
**Vuetify Version:** 3.7.6
**Vue Version:** 3.5.13
**Browsers:** Chrome 131.0.0.0
**OS:** Linux x86_64

### Steps to reproduce
Add a v-layout
Add v-navigation-drawer with permanent and location="top" props
Add a v-main component with some content

### Expected Behavior
The v-main content should be right below the v-navigation-drawer, v-main should have a padding top of around ~90px

### Actual Behavior
The v-main content is a lot further down v-navigation-drawer, it has a top padding of 320px

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNp1kcFOwzAMhl/FypnQCW5TqcQ7cKMcssa0YW4SJU63CvHuuO20gRCXWJZ//5/tvH6qnLrqOcb7qaDaq5pxjGQYm9YD1JMmM4fC0JHJ+alVKRRv0cIl6tG2apWuYhOjPpgEXaCQRN0nnDW5fmD0+qFVwI4JpSBAcp1hFzxIg1TeBdpAXV1Nbq7eTK5ftdomc8IEFLZeceIQpfsX0Jp0vPAiptF49LyZATThuDD+WG71G3M0zkPmeZ12dF4PuKyxh8fdLp6vOwO8DC4D45khD6GQhQPCR8kskcIJ/kUtUyyQ7c6SbIeWtK5+/IH6uvOFaH3evgEG4pHt)

### Comments
Basically i was trying to copy this design from the vuetify page, but ran into the issue on smaller screens
![image](https://github.com/user-attachments/assets/85165304-01fe-41c6-a85c-23d0608975c4)
